spec/system/jobseekers_can_change_email_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "rails_helper"
+
+RSpec.describe "Jobseekers can change email" do
+  let(:jobseeker) { create(:jobseeker, email: "old@email.com", password: "password") }
+  let(:created_jobseeker) { Jobseeker.first }
+
+  before do
+    allow(JobseekerAccountsFeature).to receive(:enabled?).and_return(true)
+    login_as(jobseeker, scope: :jobseeker)
+    visit edit_jobseeker_registration_path
+  end
+
+  describe "updating email" do
+    it "validates and submits the form, sends confirmation and notification emails and redirects to check your email page" do
+      update_jobseeker_email(jobseeker.email, jobseeker.password)
+      expect(page).to have_content("There is a problem")
+
+      expect { update_jobseeker_email("new@email.com", jobseeker.password) }.to change { delivered_emails.count }.by(2)
+      expect(delivered_emails.first.subject).to eq(I18n.t("jobseeker_mailer.email_changed.subject"))
+      expect(delivered_emails.first.to.first).to eq(jobseeker.email)
+      expect(delivered_emails.second.subject).to eq(I18n.t("jobseeker_mailer.confirmation_instructions.reconfirmation.subject"))
+      expect(delivered_emails.second.to.first).to eq("new@email.com")
+      expect(current_path).to eql(jobseekers_check_your_email_path)
+    end
+  end
+
+  describe "confirming email" do
+    it "confirms the email, updates the email, and redirects to saved_jobs page" do
+      update_jobseeker_email("new@email.com", jobseeker.password)
+      confirm_email_address
```

Comment:
Should we use `visit first_link_from_last_mail` here, so that we can be sure the link and token are included in the email?

Reply:
Going to raise a separate PR for implementing the use of that helper across jobseeker system specs